cmd/zetaclientd/debug.go

```diff
@@ -170,10 +170,10 @@ func debugCmd(_ *cobra.Command, args []string) error {
 			fmt.Println("CoinType not detected")
 		}
 		fmt.Println("CoinType : ", coinType)
-	} else if chains.IsBitcoinChain(chain.ChainId, appContext.GetAdditionalChains()) {
+	} else if chain.IsUTXO() {
```

Comment:
Why changing from IsBitcoin? There could be potentially other blcokchain on UTXO model that are different?

Reply:
I think that we want to check a chain by a "type", not by a particular name. That's why we have `IsEVM(chainID)` and not `IsEthereum(chainID)`. And indeed if we want to add more BTC-like chains, `IsBitcoin(chainID)` would be confusing